### DISCUSSION TITLE
Import existing Sub CA

### DISCUSSION THREAD

**Discussion by mixman68** [Keyfactor/ejbca-ce#636] [Q&A]
Hello,

I try to import a Sub CA into my EJBCA for issuing certificate and I have this error

Cannot import CA REDACTED: Certificate is not self-signed. Check certificate chain in PKCS#12

How I can import SubCA without importing RootCA private key ?

Thanks

**Comment by primetomas** [comment]
How do you do the import? You need to import the Root CA certificate, but the private key is not needed. The imported Root CA Certificate it will show up as "External CA" in EJBCA.
